lib/aggregations/class-taxonomy.php

```diff
@@ -17,6 +17,16 @@
  */
 class Taxonomy extends Aggregation {
 
+	/**
+	 * The mode by which comparisons are made in filtering. If set to AND, all
+	 * specified terms must match. If set to OR, only one of the specified
+	 * terms needs to match. Defaults to AND so that selecting additional terms
+	 * makes the result set smaller, not larger.
+	 *
+	 * @var string
+	 */
+	protected string $compare = 'AND';
```

Comment:
is there a way for this to be injected into the class from the controller? In other words, to make this more flexible, could this be handled via config instead of needing to extend the class to change this value?

Reply:
It can be configured by passing the `$args` property. Which reminds me, I need to update the wiki...

Reply:
Actually, I'm going to rename this to `relation` to match how WP_Query works.

Reply:
It's in the docs now: https://github.com/alleyinteractive/elasticsearch-extensions/wiki/Configuration-Options#enable_taxonomy_aggregation

Reply:
awesome, you're the best!